pygeoapi/django_/views.py

```diff
@@ -551,6 +535,7 @@ def admin_config_resource(request: HttpRequest,
                               resource_id)
 
 
+# TODO: remove this when all views have been refactored
```

Comment:
I guess this is the same type of comment as I left on both the starlette and flask modules - I'm not sure if this is in scope of this PR but it seems to me that it should be.

Leaving it like this implies that we keep some pretty similar functions around and use them in similar places, which will likely cause confusion and will introduce subtle bugs. I would be in favor of having this PR completely refactor the code so as to not use the older implementations and remove them.

Reply:
This is also related to https://github.com/geopython/pygeoapi/pull/1405#discussion_r1545755291